### PR TITLE
Improvements to exporter image, drone CI 

### PR DESCRIPTION
**Pull request by HarrisonWAffel**
## Relates to: https://github.com/rancher/windows/issues/223

+ Removes slack integration steps from drone.yaml, from what I can tell this integration isn't used anymore 
+ Updates CI scripts 
     + use `ltsc2019` tags to be inline with how windows images are tagged throughout  rancher 
       + It should be noted that both `1809` and `ltsc2019` tags have the exact same contents
     + remove hyper-v isolation build condition, as all official builds will come from CI, which ensures that the host id and release ID windows versions will always match 
+ Update the docker file 
    + Removes invalid empty separators resolving several docker build warnings 
    + Removed the use of `entry.ps1`, as the exporter dependency on wins is being removed. The entry point is now just invoking the downloaded binary directly with the same arguments passed by the rancher-monitoring chart 
    +  Updated the `nanoserver` stage as we no longer need to run powershell scripts, from my testing I can see that this reduces final image size by ~100MB
+ When building the image in environments the version cannot be detected, automatically set the version to `dev` 

Changing the Dockerfile to no longer invoke wins would be considered a breaking change when using this image in rancher monitoring (as the chart needs to also be changed), so we should cut a new release of this in the `v0.1.x` line

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -190,39 +157,23 @@ platform:
...
     exclude:
-    - promote
+      - promote
 
 depends_on:
-- windows-1809
-- windows-2022
\ No newline at end of file
+  - windows-1809
+  - windows-2022
```
Comment (aiyengar2) [inline .drone.yml]: nit: add newline

File: .drone.yml [hunk 2]
```diff
@@ -172,8 +173,8 @@ steps:
...
     exclude:
-      - promote
+    - promote
 
 depends_on:
-  - windows-1809
-  - windows-2022
\ No newline at end of file
+- windows-1809
+- windows-2022
```
Comment (aiyengar2) [inline .drone.yml]: nit: newline

File: .drone.yml [hunk 3]
```diff
@@ -19,7 +19,7 @@ steps:
     settings:
       dry_run: true
       build_args:
-        - SERVER_VERSION=ltsc2019
+        - SERVER_VERSION=1809
```
Comment (aiyengar2) [inline .drone.yml]: why are we reverting to 1809?

File: .drone.yml [hunk 4]
```diff
@@ -41,7 +41,7 @@ steps:
     image: plugins/docker
     settings:
       build_args:
-        - SERVER_VERSION=ltsc2019
+        - SERVER_VERSION=1809
```
Comment (aiyengar2) [inline .drone.yml]: same question here

**Review by aiyengar2 — COMMENTED** [review]
when this is merged, it should be cut as a v0.1.0 since this version of our image will definitely not work with `rancher/wins`.

File: .drone.yml (hunk 3 above)
Comment (HarrisonWAffel) [reply .drone.yml]: Good catch, my IDE keeps auto formatting this file incorrectly. I must have copy+pasted the old format and forgot to update this line

**Review by aiyengar2 — APPROVED** [review]

**Pull request merged**